packages/framer-motion/src/render/html/utils/make-none-animatable.ts

```diff
@@ -0,0 +1,33 @@
+import { getAnimatableNone } from "../../dom/value-types/animatable-none"
+import { UnresolvedKeyframes } from "../../utils/KeyframesResolver"
+
+export function makeNoneKeyframesAnimatable(
+    unresolvedKeyframes: UnresolvedKeyframes<string | number>,
+    noneKeyframeIndexes: number[],
+    name?: string
+) {
+    /**
+     * If we detected "none"-equivalent keyframes, we need to find a template
+     */
+    let i = 0
+    let animatableTemplate: string | undefined = undefined
+    while (i < unresolvedKeyframes.length && !animatableTemplate) {
+        if (
+            typeof unresolvedKeyframes[i] === "string" &&
+            unresolvedKeyframes[i] !== "none" &&
+            unresolvedKeyframes[i] !== "0"
+        ) {
+            animatableTemplate = unresolvedKeyframes[i] as string
+        }
+        i++
+    }
+
+    if (animatableTemplate && name) {
+        for (const noneIndex of noneKeyframeIndexes) {
+            unresolvedKeyframes[noneIndex] = getAnimatableNone(
+                name,
+                animatableTemplate
+            )
+        }
+    }
+}
```

Comment:
I'm having a hard time understanding what this is doing. What is the template, just some keyframe value like a color string or numeric value?